packages/gatsby-plugin-sass/src/__tests__/gatsby-node.js

```diff
@@ -75,15 +79,6 @@ describe(`gatsby-plugin-sass`, () => {
             const config = {
               loader: jest.fn(),
               merge: jest.fn(),
-              resolve: jest.fn(() => {return {
-                plugins: [
-                  new function ExtractTextPlugin() {
-                    this.extract = (...args) => {return {
-                      extractTextCalledWithArgs: args,
-                    }}
-                  }(),
-                ],
-              }}),
```

Comment:
No longer using the `resolve` method of `webpack-configurator` in this plugin, so this can be removed.